indy_node/test/state_proof/test_state_proof_for_get_requests.py

```diff
@@ -176,3 +185,112 @@ def test_state_proof_returned_for_get_claim_def(looper,
+    assert result[TXN_TIME]
+    check_valid_proof(result)
+
+
+def check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, timestamp_fr, timestamp_to,
+                    sdk_pool_handle, revoc_reg_entry_data, check_data=True):
+    req = looper.loop.run_until_complete(build_get_revoc_reg_delta_request(
+        sdk_wallet_client[1], get_revoc_reg_def_id(sdk_wallet_steward[1], revoc_reg_def[0]), timestamp_fr,
+        timestamp_to))
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_client, req)])
+
+    if check_data:
+        result = rep[0][1]['result']
+        expected_data = revoc_reg_entry_data
+        data = result.get(DATA)['value']['accum_to']
+
+        del data['seqNo']
+        del data['txnTime']
+        assert DATA in result
+        assert data
+        assert data == expected_data
+        assert result[TXN_TIME]
+        check_valid_proof(result)
+
+
+def test_state_proof_returned_for_get_revoc_reg_delta(looper,
+                                                      nodeSetWithOneNodeResponding,
+                                                      sdk_wallet_steward,
+                                                      sdk_pool_handle,
+                                                      sdk_wallet_client,
+                                                      send_revoc_reg_entry):
+    revoc_reg_def = send_revoc_reg_entry[0]
+    revoc_reg_entry_data = send_revoc_reg_entry[1][0]['operation']
+    timestamp = send_revoc_reg_entry[1][1]['result']['txnMetadata']['txnTime']
+
+    # TODO: Uncomment when libindy version updated
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, None, timestamp + 1,
+    #                 sdk_pool_handle, revoc_reg_entry_data)
+    #
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, None, timestamp - 1,
+    #                 sdk_pool_handle, revoc_reg_entry_data, False)
+
+    # TODO: INDY-2115
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, timestamp - 2, timestamp - 1,
+    #                 sdk_pool_handle, revoc_reg_entry_data, False)
+
+    # TODO: Uncomment when libindy version updated
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, timestamp - 1, timestamp + 1,
+    #                 sdk_pool_handle, revoc_reg_entry_data)
+
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, timestamp + 1, timestamp + 2,
+    #                 sdk_pool_handle, revoc_reg_entry_data)
+
+    # TODO: INDY-2115
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, None, timestamp - 999,
+    #                 sdk_pool_handle, revoc_reg_entry_data)
+
+    # TODO: Uncomment when libindy version updated
+    # check_get_delta(looper, sdk_wallet_client, sdk_wallet_steward, revoc_reg_def, timestamp - 1000, timestamp - 999,
```

Comment:
It would be great to have a test where `from` is greater than `to`.

Reply:
ok